docs/03_tutorials/21_tic-tac-toe-game-smart-contract-Testnet.md

```diff
@@ -574,7 +576,7 @@ The create action takes two parameters, the "challenger" and the "host". The req
 
 Sign the push action with ‘mcazyfujecke@active’, the host of the game.
```

Comment:
\`mcazyfujecke@active\` (multiple instances below)

Reply:
see slack for comment about using lots of ``